tests/conftest.py

```diff
@@ -50,3 +52,10 @@ def tls_ca_certificate_private_key_encrypted_path(tls_certificate_authority):
 def tls_certificate_pem_path(tls_certificate):
     with tls_certificate.private_key_and_cert_chain_pem.tempfile() as cert_pem:
         yield cert_pem
+
+
+@pytest.fixture
+def ssl_ctx(tls_certificate):
```

Comment:
(Nit) What do you think about naming it `tls_ca_ssl_context`, for consistency with other TLS fixtures that start with `tls_ca_`?

Reply:
Indeed ! done